docs/dynamic-property-configuration.adoc

```diff
@@ -80,6 +80,53 @@ public class CentralDogmaSupplierMain {
 }
 ----
 
+== YAML Support
```

Comment:
Let me update document after https://github.com/line/decaton/pull/263 is merged into master. We can use jsonschema for YAML too.

Reply:
resolved